featherbb/Model/Forum.php

```diff
@@ -201,9 +201,15 @@ public function print_topics($forum_id, $sort_by, $start_from)
                     $cur_topic['subject'] = Utils::censor($cur_topic['subject']);
                 }
 
-                if ($cur_topic['sticky'] == '1') {
+                if ($cur_topic['sticky'] == '1' && $cur_topic['closed'] == '0') {
                     $cur_topic['item_status'] .= ' isticky';
```

Comment:
Maybe you could refactor the code using a big if ($cur_topic['sticky'] == '1') having the $cur_topic['item_status'] line and do the conditions below?